src/io/flutter/view/FlutterView.java

```diff
@@ -317,7 +316,7 @@ protected void perform(AnActionEvent event) {
   }
 }
 
-abstract class AbstractFlutterAction extends DumbAwareAction {
+abstract class AbstractFlutterAction extends AnAction {
```

Comment:
Yeah, I think dumbaware is about things that need to have info from the index available before they run, which I don't think applies to the FlutterView.